### ISSUE TITLE
Windows exe icon won't change

### ISSUE THREAD

**Issue by number435398** [litecoin-project/litecoin#544]
When I refer to the icon, i'm referring to the black and white "L" that is tied to the windows executable.  I am not referring to the icon that is subsequently displayed after the program has opened.

I have gone so far as to remove the "L" icon files from the whole project and replaced them with other images but none of them show.  It continues to compile with the black and white "L" icon even when that image file doesn't even exist on the Linux VM I'm compiling on.

**Comment by thrasher-** [comment]
In line with https://github.com/litecoin-project/litecoin/issues/542, we do not assist with altcoin development. Please use another discussion medium for that, this is related to issues specifically related to an umodified version of Litecoin Core.

**Comment by number435398** [comment]
I didn't ask for assistance with alt-coin development.  The icon image seems to be derived from an unknown source, how shall you update or someone offer an update to the icon image within the boundaries of the litecoin wallet development if the code is buggy in association with that?  

Even if it was an alt-coin issue, it causes you no harm to leave this for others to examine if they so wish; without closing it.

**Comment by thrasher-** [comment]
Judging by your previous issue, you specifically stated that you were making changes for an altcoin and then post this. Why are you trying to change the icon? Sure, I made an assumption, but I just joined the two together. The image isn't sourced by an unknown location, it's referenced in the source code. With alt-coin development for others, if they change consensus critical code and don't understand what is happening or don't know how to update an icon, then honestly this is the wrong place. I don't mean to be rude, but we're in a field where a mistake can cost others serious real world consequences and thus we refrain from helping with alt coin development.

**Comment by number435398** [comment]
I'm not asking for help regarding alt-coin development.  Were I doing so, I assure you, I would have presented many more questions that did not have sufficient relationship to litecoin.  But I didn't because I know any issues I have must relate to litecoin or it is out of your scope.

> if they change consensus critical code and don't understand what is happening or don't know how to update an icon

You have, once again, made a condescending assumption.  My other concern wasn't consensus changing, though I did reference it as supplemental information, the issue was regarding the debug.log output.  That was it.  If you really want to know what the source of the block that was being rejected; it wasn't about changing code, it was about not changing it.  I hadn't changed the maximum permissible adjustment of the difficulty from litecoin to my coin, ala **_pow.cpp_**:
```
if (nActualTimespan < params.nPowTargetTimespan/4)
    nActualTimespan = params.nPowTargetTimespan/4;
if (nActualTimespan > params.nPowTargetTimespan*4)
    nActualTimespan = params.nPowTargetTimespan*4;
```

The issue with the icon; I know how to change an icon.  That's the point.  Though I just took the clean litecoin git, changed the icon files and suddenly it worked.  So it would seem I would have to abandon this issue at this point anyhow for being unable to recreate it.

**Comment by kcw-grunt** [comment]
While I don’t work primarily work on litecoin-core @number435398,  I do work with @thrasher and @losh on Litecoin projects.  They are being quite polite and you can’t expect them to debug your altcoin...they are busy..very busy.  Please understand you would get much more traction elsewhere and it will not distract us from what we are doing...making Litecoin more awesome than it already is.

**Comment by number435398** [comment]
@kcw-grunt wow, someone else accusing me of expecting you guys to debug my altcoin (which I did not do).  More extreme, and untrue claims.  If they're busy, that's fine, don't respond then.  Fair enough by me.  I've already resolved the presented issues on my own (except the debug.log).  And, no, inappropriately denigrating someone and accusing them of wild things like "not knowing what's going on" when my focus was the debug.log is not objectively "polite".  Sad that so many programmers in the industry lose track of what is polite.  Too much time staring at screen, perhaps?  Maybe I should describe it differently:

`if (extrapolation || description_to_be_conveyed > the__definitive_objective_meaning_of_words_used_by_person)
{
    if (extrapolation || description_to_be_conveyed == negative_description_of_person)
    {
        bool this_is_rude = true
        bool this_is_polite = false
        bool this_is_quite_polite = false
        bool is_merely_an_emotional_conveyance_rather_than_logic = true
    }
}
